### PR TITLE
exports correction

### PR DESCRIPTION
**Pull request by TomRichtr**
under certain condition nodes were displayed by offset of width of node group on scale 1, which is 108px

### CODE REVIEW

File: painter/index.js [hunk 1]
```diff
@@ -1,261 +1,327 @@
...
-            let iconsEle = tpc.querySelectorAll(".icons > span");
-            for (let i = 0; i < iconsEle.length; i++) {
-                let icon = iconsEle[i];
-                let iconRect = icon.getBoundingClientRect();
-                icons += `
+/**
+ * Sets svg attributes of node for export
+ * @param {Number} offset - corrective adjustment of horizonal position
+ */
+function PrimaryToSvg(primaryNode, offset = 0) {
```
Comment (VaclavS16) [inline painter/index.js:134]: chybi jsdoc k primaryNode

File: painter/index.js [hunk 2]
```diff
@@ -1,261 +1,327 @@
...
-        let a = document.createElement("a");
-        a.href = imgURL;
-        a.download = fileName || getFileName() + `.${imageType}`;
-        a.click();
-    } else {
-        return imgURL;
+/**
+ * Prepares two arrays. First array depths of left mm branches, second includes
+ * all branches.
+ * @param {Number} mm - mind map instance
```
Comment (VaclavS16) [inline painter/index.js]: tam by mel byt typ Object nebo nejaky vlastni definovany pokud je to instance

File: painter/index.js (hunk 1 above)
Comment (TomRichtr) [reply painter/index.js:134]: Upraveno

**Review by TomRichtr — COMMENTED** [review]
Upraveno

File: painter/index.js (hunk 2 above)
Comment (TomRichtr) [reply painter/index.js]: mind map instance jest object. Nechal bych to tak. :-)

**Pull request merged**